policyengine_uk/parameters/gov/hmrc/national_insurance/class_1/rates/employer.yaml

```diff
@@ -5,5 +5,7 @@ metadata:
   propagate_metadata_to_children: true
   unit: /1
   reference: https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/882271/Table-a4.pdf
+  # new rate: https://www.gov.uk/national-insurance-rates-letters
```

Comment:
Can you add this as a reference attached to the actual value? Example [here](https://github.com/PolicyEngine/policyengine-uk/blob/7e581762fee11be4191712edd5ad8837871d6198/policyengine_uk/parameters/gov/dwp/dla/mobility/lower.yaml#L12)